pkg/kernel/Dockerfile

```diff
@@ -189,16 +188,6 @@ RUN make -j "$(getconf _NPROCESSORS_ONLN)" CROSS_COMPILE="${CROSS_COMPILE_ENV}"
 # Modules
 RUN make -j "$(getconf _NPROCESSORS_ONLN)" CROSS_COMPILE="${CROSS_COMPILE_ENV}" ARCH="${KERNEL_ARCH}" INSTALL_MOD_PATH=/tmp/kernel-modules modules_install
 
-# Out-of-tree, creepy modules
```

Comment:
This creepy comment should remain since we also have rtl8821CU etc from out of tree.
(But subsequent code should be removed as you did)

Reply:
I re-added it.